src/components/SearchResults.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "../styles/searchResults.css";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line react/function-component-definition
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p className="para-fail">No results</p>;
+    // eslint-disable-next-line no-else-return
```

Comment:
Rather than adding rules like this you can use the information it gives you to fix the problem. In this case, if the `if` statement evaluates to false and the return on line 8 is not hit, the rest of the function will also not be hit meaning the function does not return anything. By removing the else you make sure that if the first if condition is not met then it will still return what comes after it as a default and stop this rule from being violated (and also mean your app will function as intended!)